ivadomed/config/config_new_loader.json

```diff
@@ -0,0 +1,88 @@
+{
+    "command": "train",
+    "gpu": 0,
+    "log_directory": "log_directory/new_loader",
+    "model_name": "model_name",
+    "debugging": true,
+    "object_detection_params": {
+        "object_detection_path": null,
+        "safety_factor": [1.0, 1.0, 1.0]
+    },
+    "loader_parameters": {
+        "bids_path": "data_example_microscopy_sem",
+        "bids_config": "ivadomed/config/config_bids.json",
```

Comment:
Does it mean this config_bids could potentially change across projects?

Reply:
Not really. This custom config is a modification to the default config of `pybids` to include microscopy data, since microscopy won't be merged with pybids for a while. It includes everything that is already in the default pybids config file + relevant entities for microscopy. It could be updated if microscopy BEP changes, or if we want to include other "non-bids" modality.
For MRI and any other "official" bids modality, the custom pybids config file is not necessary and should not be used.